### ISSUE TITLE
Could not load type 'Cassette.HostBase'

### ISSUE THREAD

**Issue by kamranayub** [andrewdavey/cassette#261]
Testing out latest beta1 and I get this error immediately after starting my site after installing:

```
Could not load type 'Cassette.HostBase' from assembly 'Cassette, Version=2.0.0.38276, Culture=neutral, PublicKeyToken=null'.

[TypeLoadException: Could not load type 'Cassette.HostBase' from assembly 'Cassette, Version=2.0.0.38276, Culture=neutral, PublicKeyToken=null'.]
   Cassette.Aspnet.CassetteHttpModule.Init(HttpApplication httpApplication) +0
   System.Web.HttpApplication.RegisterEventSubscriptionsWithIIS(IntPtr appContext, HttpContext context, MethodInfo[] handlers) +418
   System.Web.HttpApplication.InitSpecial(HttpApplicationState state, MethodInfo[] handlers, IntPtr appContext, HttpContext context) +172
   System.Web.HttpApplicationFactory.GetSpecialApplicationInstance(IntPtr appContext, HttpContext context) +336
   System.Web.Hosting.PipelineRuntime.InitializeApplication(IntPtr appContext) +300

[HttpException (0x80004005): Could not load type 'Cassette.HostBase' from assembly 'Cassette, Version=2.0.0.38276, Culture=neutral, PublicKeyToken=null'.]
   System.Web.HttpRuntime.FirstRequestInit(HttpContext context) +9798700
   System.Web.HttpRuntime.EnsureFirstRequestInit(HttpContext context) +101
   System.Web.HttpRuntime.ProcessRequestNotificationPrivate(IIS7WorkerRequest wr, HttpContext context) +251
```

I am using VS11 so I have .NET Framework 4.5 Beta installed. There must be an underlying exception I'm not seeing.

I can confirm that this happens in a brand new MVC 4 project after pulling in Cassette.AspNet.

**Comment by andrewdavey** [comment]
Can you run the Fusion Log Viewer and see if Cassette is failing to load because it can't resolve another dependency?

**Comment by kamranayub** [comment]
I will try that this weekend if I have some time. I've upgraded to VS 2012 so I'll see if the problem persists.

**Comment by kamranayub** [comment]
Hey @andrewdavey, this is fixed now with the latest build. I built locally and it works fine. I am not sure if it's simply due to me building locally using VS 2012 RC or if something has changed. Does the version out on Nuget (beta1) use the new `IConfiguration<BundleCollection>`? Because I thought I pulled down beta1 off Nuget and I was still using the old syntax and had no compile errors. As soon as I pulled the one I just built into the project, I got compilation errors.

**Update**: Scratch all that. It must have originally been an issue with old DLLs or something. I deleted the packages from the packages folder and re-got them via the Nuget official feed and everything works.
